Realm/Realm.SourceGenerator/Parser.cs

```diff
@@ -258,7 +260,7 @@ private static IEnumerable<PropertyInfo> GetProperties(ClassInfo classInfo, IEnu
             }
         }
 
-        private static PropertyTypeInfo GetPropertyTypeInfo(ClassInfo classInfo, PropertyInfo propertyInfo, IPropertySymbol propertySymbol, PropertyDeclarationSyntax propertySyntax)
```

Comment:
Same question as above.

Reply:
This method at some point uses `_generatorConfig`, that is a private (non static) variable on the class.

Reply:
Argh... sometimes I get a bit lost with the UI of GH when it does that code folding/hiding😫